README.md

```diff
@@ -1,2 +1,20 @@
 # ConvergeCodelab
+
 Android Application for retrieving Github users
+
+# Wireframes
+
+There are three wireframes in the wireframe folder displaying both static and interaction elements.
+
+**[Login mockup](https://github.com/PromasterGuru/ConvergeCodelab/blob/ft-design-mockups-158792524/wireframes/Login.png)** : Mocks the user login activity that will be used to authenticate users.
```

Comment:
Good job on the README, moreover it'll make more sense if there an image.

Reply:
> Good job on the README, moreover it'll make more sense if there an image.

Thank you. I will add the images ASAP.